_source/user-guide/kibana/index.md

```diff
@@ -8,24 +8,25 @@ tags:
   - kibana
 contributors:
   - imnotashrimp
+  - shalper
 ---
 
 When you first sign in to Logz.io, you'll find yourself at the Discover page. You can use the Discover page to filter recent logs, search for specific events, and create alerts based on your search queries.
 
-![Kibana UI in Logz.io](https://dytvr9ot2sszz.cloudfront.net/logz-docs/kibana/kibana--discover-annotated.png)
+![Kibana UI in Logz.io](https://dytvr9ot2sszz.cloudfront.net/logz-docs/kibana/kibana--discover-annotated1.png)
 
 Query bar
-: This is where you type your Elasticsearch query
+: This is where you type your search query
 
-Field and account selection
-: Choose the accounts to search and the fields that will get a table column
+Account and field selection
+: Choose the accounts to search and add fields to the preview table
 
 Histogram
-: Time-limited view of your search query
-
-Cognitive Insights
-: If Logz.io notices patterns or anomalies in your logs, it will notify you with Cognitive Insights
+: This shows you the volume of data returned for your search query. You can select a time period directly from here
```

Comment:
This is so unclear...."You can select a time period directly from here". How? I assume this is the drag functionality.